pegasus/sites.v3/code.org/public/ops-dashboard/index.haml

```diff
@@ -89,6 +89,7 @@ title: "Ops Dashboard"
         //  'remove': {method:'DELETE'},
         //  'delete': {method:'DELETE'}
         update: {method:'PUT', url: '/dashboardapi/workshops/:id'}
+        markAttendance: {method:'PUT', url: '/dashboardapi/workshops/:id'}
```

Comment:
is this just the same thing as update? if so, does it need to be another method or can you just use update?